sklearn/svm/base.py

```diff
@@ -484,6 +484,16 @@ def coef_(self):
     def _get_coef(self):
         return safe_sparse_dot(self._dual_coef_, self.support_vectors_)
 
+    @property
+    def n_support_(self):
```

Comment:
If the user calls this one before fit, they'll get an `AttributeError` on `_n_support`, which can be confusing for them. We should do that explicitly here.

We have quite a few of these properties where we manually raise the `AttributeError`, I think it makes sense to either have a decorator like `@delecate_...` or a utility function, maybe.